### PR TITLE
Add XML of Meter Indentification cluster and generated files

### PR DESCRIPTION
**Pull request by Michail-Antropov**
Add Meter Identification cluster data model, XML files by Spec (https://github.com/CHIP-Specifications/connectedhomeip-spec/tree/master) and corresponding generated files

### CODE REVIEW

File: src/app/common/templates/config-data.yaml
Comment (bzbarsky-apple) [inline src/app/common/templates/config-data.yaml:39]: This is a claim about implementation.  There is no implementation in this PR, so this part should not be here.

File: src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml:33]: This cluster does not seem to be in the spec, which makes it impossible to review this PR.

If this is based on an un-merged spec PR, the PR description here should point to that, to allow this to be reviewed.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml:33]: And I had said that at https://github.com/project-chip/connectedhomeip/pull/33922#discussion_r1640577950....

File: src/app/zap_cluster_list.json
Comment (bzbarsky-apple) [inline src/app/zap_cluster_list.json:138]: Why all the spurious changes to this file?  And this bit is clearly wrong....

I would not have expected any changes to this file at all until we are adding implementation for this cluster.

Comment (Michail-Antropov) [reply src/app/zap_cluster_list.json:138]: It is result of zap tool.
This PR has not manual work.
- alchemy converts *.adoc to *.xml
- zap tool generates other from *.xml

Comment (bzbarsky-apple) [reply src/app/zap_cluster_list.json:138]: It's possible alchemy makes modifications to this file, but where exactly did the string "METER_INDENTIFICATION_CLUSTER" come from?  Running alchemy on top of an already-modified tree can lead to weird issues like this.  I am guessing you hand-edited this json file, then ran alchemy on top of those changes.

Comment (Michail-Antropov) [reply src/app/zap_cluster_list.json:138]: Hm, than I am going to regen all with a clean slate

Comment (Michail-Antropov) [reply src/app/zap_cluster_list.json:138]: This change is result of the alchemy tool - I checked

Comment (bzbarsky-apple) [reply src/app/zap_cluster_list.json]: > This change is result of the alchemy tool - I checked

I am 100% sure that "METER_INDENTIFICATION_CLUSTER" was not the tool.  And that part is gone in the PR now, which is good.

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml:1]: Note: Please do not merge this (and specifically do not resolve this comment) until https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9245 merges in the spec.  At that point we need to compare this XML to what actually merged.

Comment (Michail-Antropov) [reply src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml:1]: Spec in master

**Review by hasty — APPROVED** [review]

File: data_model/master/clusters/MeterIdentification.xml
Comment (rbultman) [inline data_model/master/clusters/MeterIdentification.xml:3]:
```suggestion
Copyright (C) Connectivity Standards Alliance (2024). All rights reserved.

```

**Review by michaelleecowan7 — APPROVED** [review]
Approve

**Review by rbultman — APPROVED** [review]

File: data_model/master/clusters/MeterIdentification.xml
Comment (dsurnin) [reply data_model/master/clusters/MeterIdentification.xml:3]: thank you, fixing it now

File: data_model/master/clusters/MeterIdentification.xml
Comment (andy31415) [inline data_model/master/clusters/MeterIdentification.xml:3]: I expect these to be updated via spec import only. How was this updated?

Comment (andy31415) [reply data_model/master/clusters/MeterIdentification.xml:3]: And also why? Why is every line change? I would have expected them to be the same. Was this a windows vs linux newline change? please preserve newline style.

**Comment by lboue** [comment]
There's a typo in the title:
Indentification -> Identification

File: data_model/master/clusters/MeterIdentification.xml
Comment (dsurnin) [reply data_model/master/clusters/MeterIdentification.xml]: Fixed

**Comment by andy31415** [comment]
@Michail-Antropov could you resolve the merge conflicts here (basically zap-regen)? The PR now has all approvals so it should be able to go in.

**Comment by Michail-Antropov** [comment]
Meter Identification is going to update. After this PR will be reopen (or be create new).

**Pull request closed without merging**
